SUSSTR.cpp

```diff
@@ -0,0 +1,39 @@
+#include <iostream>
+
+using namespace std;
+
+int main() {
+	// your code goes here
+    int meranumber;
```

Comment:
@rahulranjan74  Please rename these variables.

Reply:
okay sure

Reply:
@ABHISHEK-AMRUTE  done changes..

Reply:
@ABHISHEK-AMRUTE  anything left to do?